pymatviz/io.py

```diff
@@ -11,6 +11,7 @@
 import matplotlib.pyplot as plt
 import plotly.graph_objects as go
 from tqdm import tqdm
+from weasyprint import HTML
```

Comment:
`weasyprint` is not installed by default and hence cannot be imported at module level.

Reply:
Didn't expect you to jump in so early...But here you are as always. Thanks for your suggestion. Yes by doing so the we can automatically generate a `requirements.txt` with `pipreqs` to include it by default? I think this would be cleaner than have some conditional import in the middle and prompt the user to install it manually. What do you think?

Reply:
It's a single-use dep. I would say not worth having every user install it if most will never need it.

Reply:
Make sense to me! I might reverse this part.